### PR TITLE
chore: introduce Connexion to update error handler

### PR DESCRIPTION
**Pull request by petrblaho**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [x] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Review by fellipeh — APPROVED** [review]
lgtm

File: kerlescan/error.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
+from connexion.lifecycle import ConnexionRequest, ConnexionResponse
 
-def handle_http_error(error):
-    json_response = jsonify({"message": error.message})
-    response = current_app.make_response(json_response)
-    response.status_code = error.status_code
-    return response
+
+def handle_http_error(request: ConnexionRequest, exc: Exception) -> ConnexionResponse:
+    return ConnexionResponse(status_code=exc.status_code, body=json.dumps({"message": exc.message}))
```
Comment (vkrizan) [inline kerlescan/error.py:7]: Isn't there a class that would return a JSON type of response? This one adds the JSON dump without a proper mime type.

Comment (RoamingNoMaD) [reply kerlescan/error.py:7]: You can give MimeType to the ConnexionResponse constructor:
```
class ConnexionResponse:
    """Connexion interface for a response."""

    def __init__(
        self,
        status_code=200,
        mimetype=None,
        body=None,
        ....
    ):

**Review by RoamingNoMaD — APPROVED** [review]
lgtm

**Review by vkrizan — APPROVED** [review]

**Pull request merged**